homeassistant/components/buienradar/sensor.py

```diff
@@ -67,584 +67,584 @@
 SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
         key="stationname",
-        name=STATIONNAME_LABEL,
+        translation_key="stationname",
     ),
     # new in json api (>1.0.0):
     SensorEntityDescription(
         key="barometerfc",
-        name="Barometer value",
+        translation_key="barometerfc",
         icon="mdi:gauge",
     ),
     # new in json api (>1.0.0):
     SensorEntityDescription(
         key="barometerfcname",
-        name="Barometer",
+        translation_key="barometerfcname",
         icon="mdi:gauge",
     ),
     # new in json api (>1.0.0):
     SensorEntityDescription(
         key="barometerfcnamenl",
-        name="Barometer",
+        translation_key="barometerfcnamenl",
         icon="mdi:gauge",
     ),
     SensorEntityDescription(
         key="condition",
-        name="Condition",
+        translation_key="condition",
     ),
     SensorEntityDescription(
         key="conditioncode",
-        name="Condition code",
+        translation_key="conditioncode",
     ),
     SensorEntityDescription(
         key="conditiondetailed",
-        name="Detailed condition",
+        translation_key="conditiondetailed",
```

Comment:
`conditiondetailed` seems to provide state translations in the new translation file. Is this a fixed/known list? If so, the sensor needs to be marked with an `enum` device class and a list of possible options needs to be provided.

Reply:
It's a list embedded in the lib. Do you have an integration which does this so I can look at that as an example?

Reply:
All integration that use `SensorDeviceClass.ENUM`